src/seeders/20230718123611-tr-report.js

```diff
@@ -0,0 +1,66 @@
+const { SCOPE_IDS } = require('@ttahub/common');
+
+const {
```

Comment:
Any need to disable audit with the seeder stuff?

Reply:
@AdamAdHocTeam  No - seeders aren't run in production, and don't populate the audit log